docs/dev/developer-guides/transactions/transactions.md

```diff
@@ -1,61 +1,71 @@
 # Transactions
 
-Transactions in Ethereum are cryptographically signed instructions by an externally owned account (an account owned by a user and not by code). These instructions are stored in the blockchain and added to a block.
-The state of the Ethereum virtual machine (EVM) changes when a transaction is initiated. A transaction can be anything from sending ether to another account to invoking the functions of a smart contract.
+From [Ethereum.org](https://ethereum.org/en/developers/docs/transactions/):
 
-## Prerequisites
+> Transactions are cryptographically signed instructions from accounts. An account will initiate a transaction to update the state of the Ethereum network. The simplest transaction is transferring ETH from one account to another. 
 
-We recommend reading the [accounts documentation](https://ethereum.org/en/developers/docs/accounts/) as preliminary material towards understanding the content on this page.
+## Transaction data on Ethereum
```

Comment:
While this is valuable, most of the fields are relevant for old (non AA) accounts. At least we need to state that AA exists and therefore other possibilities for tx fields

Reply:
added a section to clarify this ... please check

Reply:
I thought I commented elsewhere, but don't see it now. After the separation to Eth/Era, I'm not sure what is the value in describing Eth tx format

Reply:
removed, now i see the original comment